Jason/Call_Center.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+Python OOP Assignment
+
+Author: Jason Lee
+
+"""
+
+class Call(object):
+	def __init__(self, id, name, phone_num, time, reason):
+		self.id = id
+		self.name = name
+		self.phone_num = phone_num
+		self.time = time
+		self.reason = reason
+	
+	def display(self):
+		print self.id
+		print self.name
+		print self.phone_num
+		print self.time
+		print self.reason
+
+
+Call1 = Call(123,"Jason", "405-339-3086", "14:00 AM" , "emergency")
+Call2= Call(123,"Jason", "405-339-3086", "14:00 AM" , "emergency")
+Call3 = Call(123,"Jason", "405-339-3086", "14:00 AM" , "emergency")
+Call4 = Call(123,"Jason", "405-339-3086", "14:00 AM" , "emergency")
+print Call1.time
+Call1.display()
+
+class CallCenter(Call):
+	def __init__(self, *calls):
+		print calls
+		self.call_list = calls
+		self.queue_size = len(self.call_list)
+
+	def add(self,new):
+		self.call_list.append(self.new)
```

Comment:
In this line your are appending `self.new` to the self.call_list instead of the new parameter passed to the function. Is that correct?